packages/integer-field/test/validation.test.js

```diff
@@ -66,4 +67,33 @@ describe('validation', () => {
       expect(event.detail.valid).to.be.false;
     });
   });
+
+  describe('bad input', () => {
+    let input;
+
+    beforeEach(() => {
+      integerField = fixtureSync('<vaadin-integer-field></vaadin-integer-field>');
+      input = integerField.inputElement;
+      input.focus();
+    });
+
+    it('should be valid when committing a valid number', async () => {
+      await sendKeys({ type: '1' });
```

Comment:
I had no choice other than to use `sendKeys` here because setting an invalid `value` on number inputs has no effect except showing a warning.